src/Umbraco.Web/HtmlStringUtilities.cs

```diff
@@ -20,9 +20,25 @@ public sealed class HtmlStringUtilities
         /// </summary>
         /// <param name="text">The text.</param>
         /// <returns>The text with text line breaks replaced with HTML line breaks (<br/>)</returns>
+        [Obsolete("This method doesn't HTML encode the text. Use ReplaceLineBreaks instead.")]
         public HtmlString ReplaceLineBreaksForHtml(string text)
         {
-            return new HtmlString(text.Replace("\r\n", @"<br />").Replace("\n", @"<br />").Replace("\r", @"<br />"));
+            return new HtmlString(text.Replace("\r\n", @"<br />").Replace("\n", @"<br />").Replace("\r", @"<br />"));            
+        }
+        
+        /// <summary>
+        /// Html encodes the text and replaces text line breaks with HTML line breaks.
+        /// </summary>
+        /// <param name="text">The text.</param>
+        /// <returns>The html encoded text with text line breaks replaced with HTML line breaks (<br/>)</returns>
+        public HtmlString ReplaceLineBreaks(string text)
```

Comment:
```suggestion
        public IHtmlString ReplaceLineBreaks(string text)
```

Reply:
Yeah, we might as well, as that is what Umbraco uses in property value converters etc.